README.md

```diff
@@ -5,6 +5,20 @@ docker images for the [kartotherian project](https://github.com/kartotherian/kar
 They use a mixed architecture of Kartotherian and [openmaptiles](https://github.com/openmaptiles/openmaptiles)
 You need a docker version > 18.06
 
+## Workflow
```

Comment:
thanks :+1: 

maybe we could move this section after the "quick start" instructions ?

Reply:
I find it easier to miss if put after. In this case, we just give a quick overlook and that's it.